test/e2e_tests/file_test.go

```diff
@@ -555,6 +573,13 @@ func TestFileTestSuite(t *testing.T) {
 	if fileTestAdlsPtr == "true" || fileTestAdlsPtr == "True" {
 		fmt.Println("ADLS Testing...")
 		fileTest.adlsTest = true
+	} else if dataValidationFileSharePtr == "true" || dataValidationFileSharePtr == "True" {
```

Comment:
Same as previous. This should be fileTestFileSharePtr instead of dataValidationFileSharePtr